src/Objs/Event/EventEditingConfig.js

```diff
@@ -68,6 +68,33 @@ Scrivito.provideEditingConfig("Event", {
   propertiesGroups: [socialCardsPropertiesGroup, metadataPropertiesGroup],
   initialContent: {
     ...metadataInitialContent,
+    title: "Lorem Ipsum",
     body: [new SectionWidget({})],
   },
+  validations: [
+    [
+      "title",
+
+      title => {
+        if (!title) {
+          return {
+            message: "Event title must be set.",
+            severity: "error",
+          };
+        }
+      },
+    ],
+    [
+      "date",
+
+      date => {
+        if (!date) {
+          return {
+            message: "Event date is recommended.",
```

Comment:
Since this is `severity: "info"` and not `error` I propose the following message:

```suggestion
            message: "The event date is recommended.",
```

Reply:
To me, "The event date is recommended" doesn't make sense because not the date but the action of setting it is recommended. If "The event date is not set." isn't to your liking, how about "Providing the event date is recommended." or "The event date should be set."

Reply:
Since "should" is currently used for "warning", I suggest to use the following then:

```suggestion
            message: "Providing the event date is recommended.",
```

Reply:
Yes, LGTM.